### PR TITLE
Implement start of month

### PR DESCRIPTION
**Pull request by dbellettini**

### CODE REVIEW

File: spec/unit/Onebip/DateTime/UTCDateTimeTest.php [hunk 1]
```diff
@@ -559,4 +559,25 @@ public function testDiff()
...
+        $this
+            ->forAll(
+                Generator\date(
+                    new DateTime('1980-01-01'),
+                    new DateTime('2020-12-31')
+                )
+            )
+            ->then(function (DateTime $date) {
+                $date->setTimeZone(new DateTimeZone('UTC'));
+                $expected = UTCDateTime::box($date->format('Y-m-01'));
```
Comment (giorgiosironi) [inline spec/unit/Onebip/DateTime/UTCDateTimeTest.php]: assertion seems suspiciously duplicated from the production code. What about testing postconditions instead:
- the day must be 1
- the month and the year must be the same as in the input
- the hour, minute, second must be 0

**Comment by giorgiosironi** [comment]
Also, what about a Travis CI build? Or a Go one? (Travis CI suffices imho as there is no deploy)

**Comment by dbellettini** [comment]
I agree about Travis, but then we'll need access to @onebip-admin to enable it

**Comment by dbellettini** [comment]
@giorgiosironi do you like this change?

**Comment by giorgiosironi** [comment]
:+1:

**Pull request merged**